operators/operator_sdk/osdk-about.adoc

```diff
@@ -25,9 +25,39 @@ The Operator SDK is a framework that uses the link:https://github.com/kubernetes
 
 Operator authors with cluster administrator access to a Kubernetes-based cluster (such as {product-title}) can use the Operator SDK CLI to develop their own Operators based on Go, Ansible, or Helm. link:https://kubebuilder.io/[Kubebuilder] is embedded into the Operator SDK as the scaffolding solution for Go-based Operators, which means existing Kubebuilder projects can be used as is with the Operator SDK and continue to work.
 
-[NOTE]
+[id="osdk-compatibility-matrix"]
+== Operator SDK compatibility matrix
+
+
+[WARNING]
+====
+The compatibility asserts the compatibility of the operator SDK itself, it does not take into consideration the API calls done by your code.
+These must be asserted by the developer of the operator.
+====
+
+**Helm-based and Ansible-based operators compatibility**
+
+Operators based in Helm and Ansible must follow a predefined project layout. There are breaking changes between Operator SDK 0.19 and 1.3
+
+[options="header"]
+|===============================================================================================================
+|Operator SDK version | Builder image version | Compatible OpenShift version | Compatible Kubernetes API version 
+| 0.19                | 4.6                   | 4.6 and 4.7                  | 1.18, 1.19
+| 1.3                 | 4.7                   | 4.6 - 4.8                    | 1.18 - 1.20
```

Comment:
Operator SDK hasn't been updated to kubernetes 1.20 yet. So I'd say let's keep this more conservative:
```suggestion
| 1.3                 | 4.7                   | 4.6 - 4.8                    | 1.18, 1.19
```

Reply:
I have two minor concerns about this change:

1. In that row we say we support up to 4.8 will be 1.20, which means these are conflicting statements.
2. We have again the problem of knowing how can the upgrade be handled.

I think we should make it "4.6, 4.7" and "1.18, 1.19" instead, and add a note  saying:

> Currently Red Hat cannot provide an accurate statement with regards to the {product name} 4.8 and the next Operator SDK compatibility accross versions. By the time OCP 4.8 is released there will be a clear upgrade path. And this is a statement

And I don't know how the Operator SDK release cycles work, but Ideally we would want this a few weeks before the 4.8 is released, maybe by the 4.8 code freeze to name a date.